django_netjsonconfig/base/admin.py

```diff
@@ -200,6 +204,17 @@ def download_view(self, request, pk):
         return send_file(filename='{0}.tar.gz'.format(config.name),
                          contents=config_archive.getvalue())
 
+    def device_context_view(self, request, pk):
+        instance = get_object_or_404(self.model, pk=pk)
```

Comment:
use `device` instead of instance

Reply:
Closing due to https://github.com/openwisp/django-netjsonconfig/pull/138#discussion_r362201960